subsys/bluetooth/controller/Kconfig

```diff
@@ -286,5 +286,12 @@ config BT_CTLR_SDC_CX_ADV_CLOSE_ADV_EVT_ON_DENIAL
 endchoice
 endif # MPSL_CX && BT_BROADCASTER
 
+config BT_CTLR_LE_POWER_CONTROL
```

Comment:
Is this just here until the upstream PR with Kconfigs and host apis is merged? Do you plan to remove it later or are both needed?

Reply:
I will remove if i can add it to upstream. Now it is not possible as there is no code along with kconfig.  eventually there will be one kconfig for the LE Power Control feature.